resources.md

```diff
@@ -3,16 +3,40 @@ layout: page
 title: "Resources"
 ---
 
-These are various resources and readings related to machine learning and
-data-centric AI that are recommended by the course staff:
+Various resources related to machine learning and data-centric AI that we recommend.
+
+## Short Articles
 
 - [Unbiggen AI](https://spectrum.ieee.org/andrew-ng-data-centric-ai)
 - [Andrew Ng Launches A Campaign For Data-Centric AI](https://www.forbes.com/sites/gilpress/2021/06/16/andrew-ng-launches-a-campaign-for-data-centric-ai/)
-- [Human-in-the-Loop Machine Learning: Active Learning and Annotation for Human-centered AI](https://books.google.com/books/about/Human_in_the_Loop_Machine_Learning.html?id=LCh0zQEACAAJ)
 - [Tips for a Data-Centric AI Approach](https://landing.ai/tips-for-a-data-centric-ai-approach/)
 - [Making AI Work with Small Data](https://landing.ai/making-ai-work-with-small-data-2/)
 - [Boosting Model Performance through Error Analysis](https://landing.ai/boosting-model-performance-through-error-analysis/)
+- [Data-Centric Approach vs Model-Centric Approach in Machine Learning](https://neptune.ai/blog/data-centric-vs-model-centric-machine-learning)
+- [Intro to Exploratory data analysis (EDA) in Python](https://www.kaggle.com/code/imoore/intro-to-exploratory-data-analysis-eda-in-python)
+
+## Papers
+
 - [A Data Quality-Driven View of MLOps](https://arxiv.org/abs/2102.07750)
```

Comment:
thoughts on adding confident learning paper + label errors paper since we're covering those results in multiple places in the course and advocate for data-centric algorithms over model-centric (only in the label errors / classification setting)

Reply:
I think the resources section should have the more "overview" type papers that don't fit in to any particular lecture, and each lecture should have a "References" section at the end of the lecture notes (so the label errors paper would go in this section in the label errors lecture).

Reply:
ya that is the structure I'm following for my lectures